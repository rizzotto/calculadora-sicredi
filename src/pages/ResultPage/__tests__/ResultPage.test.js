import React from "react";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";

import store from "../../../providers/Store";
import ResultPage from "../index";

it("renders ResultPage correctly", () => {
    const tree = renderer
        .create(
            <Provider store={store}>
                <ResultPage />
            </Provider>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
