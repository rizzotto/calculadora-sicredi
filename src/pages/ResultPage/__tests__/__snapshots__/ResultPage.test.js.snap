// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ResultPage correctly 1`] = `
<div
  className="makeStyles-container-1"
>
  <div
    className="makeStyles-bar-2"
  >
    <div
      className="makeStyles-backButton-3"
      onClick={[Function]}
    >
      <svg>
        Back.svg
      </svg>
      <div
        className="makeStyles-voltar-4"
      >
        Voltar
      </div>
    </div>
    <svg
      height={64}
      width={64}
    >
      Logo.svg
    </svg>
  </div>
  <div
    className="makeStyles-content-5 makeStyles-content-9"
  >
    <div
      className="makeStyles-title-6 makeStyles-title-10"
    >
      Distribuição de Resultados
    </div>
    <div
      className="makeStyles-coin-7 makeStyles-coin-11"
    >
      R$
       
      <div
        className="makeStyles-value-8 makeStyles-value-12"
      >
        0.00
      </div>
    </div>
  </div>
</div>
`;
